<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="AHB-LAP" name="com.qst.ohrm.test.OrangeHRMLoginTest" tests="2" failures="1" timestamp="19 Jun 2018 10:27:00 GMT" time="38.648" errors="0">
  <testcase name="validateLoginFailTest" time="34.446" classname="com.qst.ohrm.test.OrangeHRMLoginTest">
    <failure type="java.lang.AssertionError" message="Test Failed">
      <![CDATA[java.lang.AssertionError: Test Failed
at org.testng.Assert.fail(Assert.java:96)
at com.qst.ohrm.utils.Log.fail(Log.java:101)
at com.qst.ohrm.page.OrangeHRMLoginPage.loginToOrangeHRM(OrangeHRMLoginPage.java:52)
at com.qst.ohrm.test.OrangeHRMLoginTest.validateLoginFailTest(OrangeHRMLoginTest.java:61)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
    </failure>
  </testcase> <!-- validateLoginFailTest -->
  <testcase name="validateLoginTest" time="4.202" classname="com.qst.ohrm.test.OrangeHRMLoginTest"/>
</testsuite> <!-- com.qst.ohrm.test.OrangeHRMLoginTest -->
